src/app/pages/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,71 @@
+import './LandingPage.scss'
+import { LinkProps } from '@mui/material/Link'
+import Button from '@mui/material/Button'
+import AcUnitIcon from '@mui/icons-material/AcUnit'
+import FacebookIcon from '@mui/icons-material/Facebook'
+import TwitterIcon from '@mui/icons-material/Twitter'
+import LinkedInIcon from '@mui/icons-material/LinkedIn'
+import InstagramIcon from '@mui/icons-material/Instagram'
+
+export function LandingPage() {
+  return (
+    <>
```

Comment:
Since we have TS, shall we export it as `export const LandingPage: React.FC = () => {
         return (
            <React.Fragment>
             ................................
           </React.Fragment>
    )
]`
?

Reply:
Thanks github for deleting my formatting :)